Emerging_Threats/guloader.yml

```
@@ -0,0 +1,155 @@
+version: 3
+resources:
+  replicant:
+    - exfil
+Rules:
+  'GuLoader #001':
+    namespace: general
+    detect:
+      event: DNS_REQUEST
+      op: is
+      path: event/DOMAIN_NAME
+      value: ip130.ip-5-39-3.eu
+    respond:
+      - action: report
+        name: guloader-2021-12-02
+  'GuLoader #002':
+    namespace: general
+    detect:
+      event: DNS_REQUEST
+      op: is
+      path: event/DOMAIN_NAME
+      value: nowautomation.com
+    respond:
+      - action: report
+        name: guloader-2021-12-02
+  'GuLoader #003':
+    namespace: general
+    detect:
+      event: NETWORK_CONNECTIONS
+      op: is
+      path: event/NETWORK_ACTIVITY/?/DESTINATION/IP_ADDRESS
+      value: 99.83.154.118
+    respond:
+      - action: report
+        name: guloader-2021-12-02
+'GuLoader #004':
+  namespace: general
+  detect:
+    event: ONGOING_IDENTITY
+    op: ends with
+    path: event/HASH
+    value: c00b66ef61df2012b269bca3e60b301478641292948f1cac579096603ad67f98
+  respond:
+    - action: report
+      name: 'signed-guloader-2021-12-02 (https://attack.mitre.org/software/S0561/)'
+'GuLoader #005':
+  namespace: general
+  detect:
+    event: ONGOING_IDENTITY
+    op: ends with
+    path: event/HASH
+    value: 4366a0e113d168f2809a4a1983c2198ec874b89af0bdfe465e753d409c85c51c
+  respond:
+    - action: report
+      name: 'loki-guloader-2021-12-02 (https://attack.mitre.org/software/S0561/)'
+'GuLoader #006':
+  namespace: general
+  detect:
+    event: ONGOING_IDENTITY
```

Those should be targeting `CODE_IDENTITY`. The `ONGOING_IDENTITY` is not reported to the cloud by default.
```suggestion
    event: CODE_IDENTITY
```

---

I agree,
I was using the as reference the previous template pushed/reviewed on the repo:
https://github.com/refractionPOINT/rules/blob/617b4272c7a07cfb20e3dc1c7aa8a0b1fcdc3c71/Emerging_Threats/blackmatter_ransomware.yml#L39